src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -1051,6 +1051,17 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
                     // this whitelist condition is temporarily and to be relaxed soon.
                     && !fused_node->is_type<fully_connected>())
                     return;
+
+                // We need to broadcast batch dimention for OneDNN binary add fusions.
+                // But, it's not possible for dynamic layers.
+                #ifdef ENABLE_ONEDNN_FOR_GPU
+                    if ((fused_node->is_type<gemm>() || fused_node->is_type<fully_connected>())
```

Comment:
Won't it prevent all gemm/fc+eltwise fusions? Looks too restrictive to me

Reply:
This is an initial draft for checking the perf impact and there was an impact. Checking another w/a to add broadcast always & skip when needed.

Reply:
Why you just don't use unfusing feature?

Reply:
oh... I'll also try to use the unfusing feature. Thank you!

Reply:
@vladimir-paramuzov I implemented it to use the unfusing feature as you suggested in https://github.com/openvinotoolkit/openvino/pull/26284. Thank you.